shipyard_blueprints/postgresql/shipyard_postgresql/postgres.py

```diff
@@ -1,15 +1,26 @@
-from shipyard_templates import Database
```

Comment:
The overall comment to the below code is the error handling is very broad. Ideally a there should be handling of more specific errors

Reply:
Interested to know your thoughts here and possible improvements. There are (in my opinion) too many possible errors to handle individually (https://docs.sqlalchemy.org/en/20/errors.html#connections-and-transactions) so I opted for more generalized error handling relying on the error message (which in my experience is often informative enough). Each method more or less has its own custom exception which spits out the message from API. I've tried to strike the balance between pragmatism and utility for end user, so if I've missed something here or you have possible improvements, I'd very much welcome that because it would not only apply to this but also to the other database blueprints